pipeline.go

```diff
@@ -9,6 +9,17 @@ import (
 
 type pipelineExecer func([]Cmder) error
 
+type Pipelineable interface {
```

Comment:
Can you please rename this to `Pipeliner`? I think it makes some sense and is more idiomatic.

`Cmdable` name was chosen because `Cmder` means something completely different...